packages/swap/src/providers/okx/index.ts

```
@@ -0,0 +1,927 @@
+              return result;
+            } catch (modifyError) {
+              // Fallback to original transaction
+              const reserializedBuffer = transaction.serialize();
+              const result = Buffer.from(reserializedBuffer).toString("base64");
+              return result;
+            }
+          } catch (versionedError) {
+            throw versionedError; // Skip legacy fallback, it's broken
+          }
+        } catch (strategyError) {
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)